test/ci/README-ci-test-trigger.txt

```diff
@@ -0,0 +1,4 @@
+This is a file being used to trigger tests of the CI system.  It will never be
+committed to master, and will be removed when the testing is done.
```

Comment:
This file feels sad :'( and sees no purpose in life now, because of Chris's absolute refusal to commit it 
:-P